src/test/java/seedu/address/logic/commands/AddDoctorCommandTest.java

```diff
@@ -88,6 +91,18 @@ public void toStringMethod() {
         assertEquals(expected, addDoctorCommand.toString());
     }
 
+    @Test
+    public void hashcode() {
+        AddDoctorCommand command = new AddDoctorCommand(ALICE);
+
+        // same value -> returns same hashcode
+        assertEquals(command.hashCode(), Objects.hash(ALICE));
+
+        // different value -> returns different hashcode
+        assertNotEquals(command.hashCode(), Objects.hash(BOB));
```

Comment:
actually different values giving the same hashcode doesnt violate lsp. The expected behaviour is a.equals(b) -> a.hashcode() == b.hashcode();